sdcm/sct_runner.py

```diff
@@ -955,14 +1145,13 @@ def clean_sct_runners(test_status: Optional[str] = None, test_runner_ip: str = N
                 LOGGER.warning("Value of `keep' tag is invalid: %s", exc)
         if dry_run:
             LOGGER.info("Skip %s because of dry-run", sct_runner_info)
-            continue
+            return
```

Comment:
should continue here too, so dry run behaves like in normal run

Reply:
Fixed